src/MALA.jl

```diff
@@ -0,0 +1,103 @@
+using ForwardDiff: gradient!
+using DiffResults: GradientResult, value, gradient
+
+struct MALA{D} <: MHSampler
+    proposal::D
+end
+
+
+MALAMH(d) = MALA(RandomWalkProposal(d))
+
+
+struct Transition_w∇{T<:Union{Vector, Real, NamedTuple}, L<:Real, G<:Union{Vector, Real, NamedTuple}}
```

Comment:
Unicode characters are difficult for some to type, I'd prefer something like this instead:

```suggestion
struct GradientTransition{T<:Union{Vector, Real, NamedTuple}, L<:Real, G<:Union{Vector, Real, NamedTuple}}
```